src/main/java/org/myrobotlab/arduino/VirtualMsg.java

```diff
@@ -1570,6 +1570,7 @@ public boolean readMsg() throws Exception {
           if (newByte > 64) {
             // TODO - send error back
             byteCount = 0;
+            System.out.println("TO BIG");

```

Comment:
we should avoid system.out.printlns....  and use log.debug/info ...  (System.out.println is good for debugging locally)

Reply:
Agreed .. this was temporary ... I was trying to isolate the root cause of the "no ack" error..
System.out - was a quick way to focus the logging to change it to error level ... filtering out all msgs except errors and this bit .. but I have removed it ..